### ISSUE TITLE
Duplicate operation on is_euclidean_depth flag

### ISSUE THREAD

**Issue by LevinJ** [OPEN-AIR-SUN/mars#88]
If I understand the codes correctly, there might be a bug in the depth loss calculation codes.

At Line 695 of nsg/models/scene_graph.py, we mutiply depth_gt by directions_norm the first time. After this operation, depth_gt is already in euclidean distances.

```
if not self.config.is_euclidean_depth:
      depth_gt = depth_gt * outputs["directions_norm"]
```

But then in the follow up sensor depth loss or mono depth loss calculation, depth_gt is multiplied again by outputs["directions_norm"] because **is_euclidean** parameter passed to them is False .

```
if self.config.depth_loss_mult > 1e-8:
                for i in range(len(outputs["weights_list"])):
                    depth_loss += self.depth_loss(
                        weights=outputs["weights_list"][i],
                        ray_samples=outputs["ray_samples_list"][i],
                        termination_depth=depth_gt,
                        predicted_depth=predicted_depth,
                        sigma=sigma,
                        directions_norm=outputs["directions_norm"],
                        is_euclidean=self.config.is_euclidean_depth,
                        depth_loss_type=self.config.depth_loss_type,
                    ) / len(outputs["weights_list"])
                 
   
            mono_depth_loss = monosdf_depth_loss(
                termination_depth=depth_gt,
                predicted_depth=predicted_depth,
                is_euclidean=self.config.is_euclidean_depth,
                directions_norm=outputs["directions_norm"],
            )
```

**Comment by xBeho1der** [comment]
Hi! Thanks for your reply! The bug has been fixed now.
